Tests/Serializer/PlainTextDeserializationVisitorFunctionalTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Smartbox\CoreBundle\Tests\Serializer;
+
+use JMS\Serializer\Naming\IdenticalPropertyNamingStrategy;
+use JMS\Serializer\SerializerBuilder;
+use Smartbox\CoreBundle\Serializer\PlainTextDeserializationVisitor;
+
+class PlainTextDeserializationVisitorFunctionalTest extends \PHPUnit\Framework\TestCase
```

Comment:
i would define the full namespace on the use part rather than in the extends of the class leaving the code to something like this (to follow the same pattern as the other `use`) 
```
use JMS\Serializer\Naming\IdenticalPropertyNamingStrategy;
use JMS\Serializer\SerializerBuilder;
use PHPUnit\Framework\TestCase;
use Smartbox\CoreBundle\Serializer\PlainTextDeserializationVisitor;

class PlainTextDeserializationVisitorFunctionalTest extends TestCase
```

Reply:
I agree but all the other tests in the test suite use the same style to declare the class. I think at some point it was the recommended way to write tests? I've seen it before.

What do you want to do? Should we fix only this one, keep it this way, or fix all the other tests?

Reply:
I will say to use the policy fix what you touch, so in this case just update this one

Reply:
Mmmm, it's just a matter of opinion or preference. Tomorrow we might think that full FQN is the way to go and this one would be "wrong", while the others are "correct". I still think it is better to have consistency, specially on trivial things like this.

Fixed it your way, anyway

Reply:
This could be something a CS fixer could fix to an standard way. Nowadays I don't see cases where the full qualified name is used in the extends. A similar example of this could be the definition of array between `array()` and `[]`. Old code would have `array()` but if you are implementing new code you would usually use `[]. The code during X time would have both references until you take specific time to move to the new way. or another one is how to declare a service in Symfony. Early in Symfony was considered to define the class names in parameters and the pass the parameter on the service definition, but at later stage they considered this was bad

Reply:
Agree, that's why I think until there's an "automated police" that enforces this, we should follow consistency rather than preference.